### DISCUSSION TITLE
GitHub dropping username/password support

### DISCUSSION THREAD

**Discussion by gilbertbw** [git-ecosystem/git-credential-manager#370] [Q&A] [answered]
I take it this project uses a PAT for GitHub, and is not impacted by GitHub [dropping support for usernames and passwords for git operations](https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/) later this year?

**Comment by derrickstolee** [comment, marked as answer]
GCM Core is a critical piece of how to authenticate with GitHub following the username/password deprecation. Using the latest GCM Core release will use the authentication mechanism that will continue after that deprecation. It integrates with multi-factor authentication according to your security requirements.

**Reply by gilbertbw** [reply]
Thanks for confirming. Not sure if it would be worth updating the linked blog post to include using GCM Core under "The following customers remain unaffected by this change"?

It might be obvious for most people, and I expected this was the case, but was not 100%.

**Reply by derrickstolee** [reply]
We will try to make sure that when the identity group sends updates that they will include information like this to help users.

**Reply by Billyright44** [reply]
Great
